src/native/tools/kube_pv_chown.cpp

```diff
@@ -49,13 +74,12 @@ main(int argc, char* argv[])
         cout << "Warning: got 0 as uid - only root will get access" << endl;
     }
 
-    if (deployment_type == "server") {
-        change_path_permissions("/log", uid);
-    } else if (deployment_type == "mongo") {
-        change_path_permissions("/mongo_data", uid);
-        change_path_permissions("/log", uid);
+    if (deployment_type == "mongo") {
+        change_path_permissions_recursive("/mongo_data", uid);
+    } else if (deployment_type == "postgres") {
+        change_path_permissions_recursive("/var/lib/pgsql", uid);
```

Comment:
if we are using an ENV to pass the EXPECTED_UID and EXPECTED_GID then use it here as well